.mailmap

```diff
@@ -204,3 +204,7 @@ Wolfgang Kerzendorf <wkerzendorf@gmail.com> Wolfgang Kerzendorf <wkerzend@Wolfga
 
 Youssef Eweis <joey070@gmail.com>
 Youssef Eweis <joey070@gmail.com> Youssef Eweis <Youssef15015@users.noreply.github.com>
+
+Rohith Varma Buddaraju <rohith.varma.buddaraju@gmail.com>
+Rohith Varma Buddaraju <rohith.varma.buddaraju@gmail.com> rohithvarma3000 <rohith.varma.buddaraju@gmail.com>
+Rohith Varma Buddaraju <rohith.varma.buddaraju@gmail.com> Rohith <rohith.varma.buddaraju@gmail.com>
```

Comment:
you need to edit `.mailmap` on just one of your PRs!

Reply:
Ok. Once one PR is merged, I will make the necessary changes.